### ISSUE TITLE
Multiple OUTPUT configurations override previous ones.

### ISSUE THREAD

**Issue by m15o** [aws/amazon-cloudwatch-logs-for-fluent-bit#5]
I tried to setup multiple OUTPUTs for different log groups using `Tag`, but both INPUTs events are pushed to the same log group.

This is my configuration:

```
[SERVICE]
    Flush 5

[INPUT]
    Name dummy
    Dummy {"message":"dummy-A"}
    Tag input-A

[INPUT]
    Name dummy
    Dummy {"message":"dummy-B"}
    Tag input-B

[OUTPUT]
    Name cloudwatch
    Match input-A
    log_group_name  input-A
    log_stream_name ${HOSTNAME}
    auto_create_group true
    region ${AWS_REGION}

[OUTPUT]
    Name cloudwatch
    Match input-B
    log_group_name  input-B
    log_stream_name ${HOSTNAME}
    auto_create_group true
    region ${AWS_REGION}
```

Log group `input-A` is empty.

Log group `input-B` got the both INPUTs:
```
07:41:28 {"message":"dummy-A"}
07:41:28 {"message":"dummy-B"}
07:41:29 {"message":"dummy-A"}
07:41:29 {"message":"dummy-B"}
07:41:30 {"message":"dummy-A"} 
07:41:30 {"message":"dummy-B"}
07:41:31 {"message":"dummy-A"}
07:41:31 {"message":"dummy-B"}
...
```

**Comment by PettitWesley** [comment]
@m15o I've reproduced this issue; can confirm its a bug. I think I know how to fix it; will try to get the fix out ASAP.

**Comment by PettitWesley** [comment]
TL;DR basically multiple instances are currently not supported.

Proximate Cause: Each call to `FLBPluginInit()` overwrites the global variable [here](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/blob/master/fluent-bit-cloudwatch.go#L32), and thus when multiple instances of the plugin are present, the final configuration is used. 

Root Cause: I didn't test any configurations with multiple instances 😐. Also, I based the plugin code off [this old example](https://github.com/fluent/fluent-bit-go/tree/master/examples/out_gstdout) (which was the only example when I started work), and did not later take a look at [this other example](https://github.com/fluent/fluent-bit-go/tree/master/examples/out_multiinstance), which shows how to do create a plugin that can support multiple instances.

Thankfully, fixing this is pretty easy.

**Comment by PettitWesley** [comment]
@m15o We've released a new version of the Fluent Bit image in Docker Hub and ECR with the fix. The images `aws-for-fluent-bit:latest` and `aws-for-fluent-bit:1.2.2` have it.
